lrpar/src/lib/parser.rs

```diff
@@ -653,21 +653,41 @@ impl<LexemeT: Lexeme<StorageT>, StorageT: Hash + PrimInt + Unsigned>
                             + 1;
                         write!(out, "\n  {}{}: ", " ".repeat(padding), i + 1).ok();
                         let mut rs_out = Vec::new();
-                        for r in rs {
-                            match r {
+
+                        let mut i = 0;
+                        while i < rs.len() {
+                            match rs[i] {
+                                ParseRepair::Delete(l) => {
```

Comment:
I think it would be helpful to have a comment here, something perhaps like,
```
// Combine adjoining `Delete` repairs".
```

Reply:
Fixed in https://github.com/softdevteam/grmtools/pull/356/commits/0b93ba106c84260820d9a0160c713669e02cce8e.